test/common/string_util_test.cpp

```diff
@@ -59,5 +59,46 @@ TEST(Common, StringToUll) {
     str = "ffff";
     ASSERT_FALSE(StringToUll(str, &out));
 }
+
+TEST(StringUtilTest, ltrim) {
+  std::array<std::array<std::string, 2>, 4> cases = {
+      std::array<std::string, 2>{"hello", "hello"},
+      std::array<std::string, 2>{"", ""}, std::array<std::string, 2>{"  ", ""},
+      std::array<std::string, 2>{"  hello", "hello"}};
+
+  for (auto &c : cases) {
+    curve::common::LTrim(c[0]);
+    EXPECT_EQ(c[0], c[1]);
+  }
+}
+
+TEST(StringUtilTest, rtrim) {
+  std::array<std::array<std::string, 2>, 4> cases = {
+      std::array<std::string, 2>{"hello", "hello"},
+      std::array<std::string, 2>{"", ""}, std::array<std::string, 2>{"  ", ""},
+      std::array<std::string, 2>{"hello  ", "hello"}};
+
+  for (auto &c : cases) {
+    curve::common::RTrim(c[0]);
+    EXPECT_EQ(c[0], c[1]);
+  }
+}
+
+TEST(StringUtilTest, trim) {
+  std::array<std::array<std::string, 2>, 6> cases = {
+      std::array<std::string, 2>{"hello", "hello"},
+      std::array<std::string, 2>{"   hello", "hello"},
+      std::array<std::string, 2>{"hello    ", "hello"},
+      std::array<std::string, 2>{"    hello  ", "hello"},
+      std::array<std::string, 2>{"    ", ""},
+      std::array<std::string, 2>{"", ""},
```

Comment:
we should add a case for middle space, such as "   S3 Browser  ", "S3 Browser", the expect trimed string is "S3 Browser".

Reply:
Fixed.